src/Tribe/Aggregator/Processes/Service_Provider.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Class Tribe__Events__Aggregator__Processes__Service_Provider
+ *
+ * @since TBD
+ */
+class Tribe__Events__Aggregator__Processes__Service_Provider extends tad_DI52_ServiceProvider {
+
+	/**
+	 * Binds and sets up implementations.
+	 *
+	 * @since TBD
+	 */
+	public function register() {
+		tribe_register( 'events-aggregator.record-items', 'Tribe__Events__Aggregator__Record__Items' );
+		tribe_register( 'events-aggregator.processes.import-events', 'Tribe__Events__Aggregator__Processes__Import_Events' );
+
+		add_filter( 'tribe_process_queues', array( $this, 'filter_tribe_process_queues' ) );
```

Comment:
Should we move the hook's into another method?

Reply:
not right now, I'm always for clean code but creating a method to call another for no added readability effect seems like speculative coding.
If there were two, though, I would have done it.